### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by mcdonnnj**
This PR updates the README to improve readability of the examples and to pass markdownlint for integration of pre-commit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,85 +1,148 @@
...
-`cyhy-data-extract` compresses the data, signs the compressed file,
-encrypts the file, and pushes the encrypted compressed file to a
-bucket in S3 using AWS creds.
+`cyhy-data-extract` retrieve and compress the specified data, sign the compressed
+file, encrypt the file, and optionally push the encrypted, compressed file to an
+S3 bucket using provided AWS credentials.
 
-`cyhy-data-retriever` pulls the file from s3, decrypts the file, and
-decompresses it.
+`cyhy-data-retriever` take a provided file (optionally stored on an S3 bucket),
```
Comment (jsf9k) [inline README.md]: "...on an S3 bucket..." --> "...in an S3 bucket..."

File: README.md [hunk 2]
```diff
@@ -1,85 +1,148 @@
...
+decrypt it, and then decompresses it to local storage.
```
Comment (jsf9k) [inline README.md]: "decompresses" --> "decompress" to keep the verb tense consistent

File: README.md [hunk 3]
```diff
@@ -1,85 +1,148 @@
...
+to remove all Python 2 requirements. Python 3 is not officially supported
+at this time.
 
-`cyhy-data-extract` can run as either Python 2 or Python 3.
+`cyhy-data-retriever` can run as either Python 2 or Python 3.
 
 To run the tool locally first install the requirements:
-```bash
+
+```console
```
Comment (jsf9k) [inline README.md:27]: Awesome!  I didn't realize that `console` was an option.

File: README.md [hunk 4]
```diff
@@ -1,85 +1,148 @@
-# Cyhy-feeds :inbox_tray: :outbox_tray:
+# Cyber Hygiene Feeds 📥 📤 #
 
+[![Build Status](https://travis-ci.com/cisagov/cyhy-feeds.svg?branch=develop)](https://travis-ci.com/cisagov/cyhy-feeds)
+[![Coverage Status](https://coveralls.io/repos/github/cisagov/cyhy-feeds/badge.svg?branch=develop)](https://coveralls.io/github/cisagov/cyhy-feeds?branch=develop)
 [![Total alerts](https://img.shields.io/lgtm/alerts/g/cisagov/cyhy-feeds.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/cisagov/cyhy-feeds/alerts/)
 [![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/cisagov/cyhy-feeds.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/cisagov/cyhy-feeds/context:python)
 
 cyhy-feeds consists of two parts; the extractor and the retriever
```
Comment (jsf9k) [inline README.md]: May as well put a period at the end of this sentence.

**Review by jsf9k — APPROVED** [review]
I made a few minor suggestions.  Please consider those.  Otherwise I think this PR is ready to merge.  Great improvements!

**Review by dav3r — APPROVED** [review]
Looks great!  I agree with @jsf9k's recommendations.  Thanks @mcdonnnj! ✌️

**Pull request merged**